certificate-policy.md

```diff
@@ -1895,20 +1885,26 @@ c. **Certificate Field:** subject:countryName (OID: 2.5.4.6)
 **Required/Optional:** Required  
 **Contents:** This field SHALL contain C=US  
 
+CA Certificate Subjects SHALL NOT include organizationalUnit unless approved by the Policy Authority.
+
+All other optional attributes, for the CA Certificate Subject fields, SHALL NOT be included.
+Optional attributes MUST NOT contain metadata such as '.', '-', and ' ' (i.e. space) characters, and/or any other indication that the value is absent, incomplete, or not applicable.  
+
+
 ### 7.1.5 Name constraints
-All Subordinate CA Certificates shall be Technically Constrained.
+All Subordinate CA Certificates SHALL be Technically Constrained.
 
-For a Subordinate CA Certificate to be considered Technically Constrained, the certificate MUST include an Extended Key Usage (EKU) extension specifying all extended key usages that the Subordinate CA Certificate is authorized to issue certificates for. The anyExtendedKeyUsage KeyPurposeId MUST NOT appear within this extension.
+For a Subordinate CA Certificate to be considered Technically Constrained, the certificate SHALL include an Extended Key Usage (EKU) extension specifying all extended key usages that the Subordinate CA Certificate is authorized to issue certificates for. The anyExtendedKeyUsage KeyPurposeId SHALL NOT appear within this extension.
 
-The Subordinate CA Certificate(s) SHALL include the id-kp-serverAuth extended key usage, then the Subordinate CA Certificate(s) SHALL include the Name Constraints X.509v3 extension with constraints on dNSName as follows:
+The Subordinate CA Certificate(s) SHALL include the id-kp-serverAuth extended key usage, and the Subordinate CA Certificate(s) SHALL include the Name Constraints X.509v3 extension with constraints on dNSName as follows:
 
 a. For each dNSName in permittedSubtrees, the CA MUST confirm that the Applicant has registered the dNSName or has been authorized by the domain registrant to act on the registrant's behalf in line with the verification practices of section 3.2.2.4.
 The Subordinate CA Certificate MUST include at least one dNSName in permittedSubtrees.  The permittedSubtrees for dNSName MUST be within the constraints of the top-level domains for:
 
  - gov (DotGov)
  - mil (DotMil)
 
-The permittedSubtrees for dNSName MUST NOT contain any other dnsName ranges outside of the the "gov" or "mil" top-level domains.
+The permittedSubtrees for dNSName MUST NOT contain any other dnsName ranges outside of the the DotGov or DotMil top-level domains.
```

Comment:
Tiny thing (and not going to hold up the merge), but I think `.gov` and `.mil` are the right terms here. `DotGov` and `DotMil` are government program names and terms, but when discussing DNS for a broader audience I think we should refer to it by the technical namespace.

Reply:
One concrete example of the confusion: the `DotGov` program also includes `.fed.us`, but the name constraints for this PKI does not include `.fed.us`.